tests/perf/cypress.config.js

```diff
@@ -0,0 +1,9 @@
+const { defineConfig } = require("cypress");
+
+module.exports = defineConfig({
+  e2e: {
+    baseUrl: 'http://localhost:3000',
+    timeout: 10000,
+    threshold: 100000
+  }
+});
```

Comment:
Does cypress 9.x support this syntax or should this be a `.json` file? cypress 10.x seems to have switched to the `js` config but we're still on 9.x.